crates/bevy_reflect/src/serde/ser.rs

```diff
@@ -932,4 +939,36 @@ mod tests {
 
         assert_eq!(expected, output);
     }
+
+    #[test]
+    fn should_return_error_if_missing_registration() {
+        let value = RangeInclusive::<f32>::new(0.0, 1.0);
+        let registry = TypeRegistry::new();
+
+        let serializer = ReflectSerializer::new(&value, &registry);
+        let error = ron::ser::to_string(&serializer).unwrap_err();
+        assert_eq!(
```

Comment:
Nit: I'm not sold on duplicating the exact error message in the tests. It becomes more brittle without really testing anything tricky 🤔 I would either cut these tests or swap to an assert! matches! pattern.

Reply:
I thought that maybe in this case it might make sense since we want to test that the errors are indeed different depending on the registrations in the type registry (missing registration vs missing type data).

However, if you think they don't really add value, I can definitely remove them!

Reply:
Nah, feel free to keep them. I don't love that we're testing for string equality, but I don't see a better way to do this.